kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/ViewRegistry.kt

```diff
@@ -13,13 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@file:Suppress("FunctionName")
+
 package com.squareup.workflow.ui
 
 import android.content.Context
 import android.view.View
 import android.view.ViewGroup
 import com.squareup.workflow.ui.backstack.BackStackContainer
-import kotlin.reflect.KClass
+
+/**
+ * [ViewBinding]s that are always available.
+ */
+internal val defaultViewBindings = ViewRegistry(
+    NamedBinding,
+    BackStackContainer,
+    ModalContainer.forAlertContainerScreen()
+)
```

Comment:
These are wired up in `WorkflowLayout` instead of being hard-coded into any particular `ViewRegistry` class.